### ISSUE TITLE
React-docs Table Headers Disappear on Mobile

### ISSUE THREAD

**Issue by redallen** [patternfly/patternfly-react#1671]
Table headers for props and css variables disappear on mobile view after #1656 ([example](https://1656-pr-patternfly-react-patternfly.surge.sh/patternfly-4/components/applicationlauncher/)).

**Comment by dgutride** [comment]
Can't repro (and react-docs are low priority since we'd like to replace gatsby and components with the config out of -org
